README.md

```diff
@@ -34,89 +34,108 @@ Loon is a repository that is meant to be deployed as a standalone docker contain
 
 ## Use And Deployment
 
-All of the necessary applications are shipped in a single docker container. In the repository, you will see two docker directories: `docker` and `docker-local`. The main deployment will use the `docker` directory. The `docker-local` directory is a separate local version of Loon which we will discuss shortly.
+All of the necessary applications are shipped in a single docker container. We use a python script called "build.py" to initiate and simplify the process.
+
+**Note that docker and Python 3+ must be installed and running in order to use the "build.py" script**
+
+We start by making a configuration file called "config.json". Below is a template:
```

Comment:
Standard is the root. It doesn't necessarily matter though as long as you don't commit it (since itll bloat the code). You can specify the location with the build script.